activesupport/lib/active_support/inflector/methods.rb

```diff
@@ -147,9 +148,9 @@ def humanize(lower_case_and_underscored_word, capitalize: true, keep_id_suffix:
       end
 
       if capitalize
-        result.sub!(/\A\w/) do |match|
```

Comment:
The initial implementation was interesting. I wonder if chaning it for `/\A([[:lower:]])/` wouldn't lead so a similar gain as it wouldn't do anything if the first letter is already capitalized.

It would allow better work with unicode letters:

```ruby
>> "étienne".sub(/\A\w/, &:upcase)
=> "étienne"
>> "étienne".sub(/\A[[:lower:]]/, &:upcase)
=> "Étienne"
```